openhtf/util/multicast.py

```diff
@@ -146,13 +147,11 @@ def send(query,
   sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
   sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, ttl)
   if local_only:
+    # Set outgoing interface to localhost to ensure no packets leave this host.
     sock.setsockopt(
         socket.IPPROTO_IP,
-        # IP_MULTICAST_IF is the 8-byte group address followed by the IP
-        # assigned to the interface on which to listen.
         socket.IP_MULTICAST_IF,
-        # Only connect to the local connection.
-        struct.pack('!4s', socket.inet_aton('127.0.0.1')))
+        struct.pack('!L', LOCALHOST_ADDRESS))
```

Comment:
The changes to send() are non-significant, I wanted to re-use the `LOCALHOST_ADDRESS` variable which was helpful for the changes to `MulticastListener`